src/020._valid_parentheses.md

```diff
@@ -67,4 +69,40 @@ class Solution(object):
                 if stack == [] or stack.pop() != dic[char]:
                     return False
         return stack == []
-```
\ No newline at end of file
+```
+
+### C++ Solutions
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```C++
+class Solution {
+public:
+    bool isValid(string s) {
+        string left = "([{";
+        string right = ")]}";       
+        stack<char> stk;
+        
+        for (auto c:s) {
+            if (left.find(c) != string::npos) {
+                stk.push(c);
+            } else {
+                if (stk.empty() || stk.top() != left[right.find(c)])
```

Comment:
```c++
                if (stk.empty() || stk.top() != left[right.find(c)]) {
                    return false;
                } else {
                    stk.pop();
                }
```

Reply:
I have corrected some format. Thank  for your careful watching! @jovan-wong